### PR TITLE
update header and footer logos

### LINKED ISSUE #1320: Update logo to new design
[SW-1801](https://jirasul.stanford.edu/jira/browse/SW-1801)

### PR DESCRIPTION
**Pull request by drh-stanford**
This PR fixes #1320. I wasn't able to convert the EPS to SVG in a way that preserves the vectors -- I'd need Photoshop, so these remain in the PNG format.

## Header

screen shot 2017-06-06 at 10 08 51 am

## Mobile Header

screen shot 2017-06-06 at 10 09 10 am

## Footer

screen shot 2017-06-06 at 10 08 58 am

### CODE REVIEW

**Comment by camillevilla** [comment]
I tried converting the EPS with Illustrator C5.1 but I can't open the files because they're not saved in a backwards compatible way. We'd need someone with Illustrator CS6.

**Comment by jkeck** [comment]
@jvine do you have the ability to resize these?

Also, do you think this would be a good opportunity to address the issue with the icon/hamburger in the header on mobile? (which existed before these changes)

**Comment by jvine** [comment]
svgs are here: https://stanford.box.com/s/sww9azcyaj1tu9uydv1dz28x4whfokza

If the hamburger menu is quick to fix, then sure. It will change later in this project, so we shouldn't invest too much time in it.

**Comment by camillevilla** [comment]
Alrighty! New logos are in and old `.png` references have been changed to `.svg.`  This should be good to merge @jkeck

**Comment by jkeck** [comment]
It looks like the stacked logo is cut-off a bit.  I loaded the image up separately just to make sure it wasn't markup or css related.

@jvine perhaps this was an export problem?

stacked-cut-off

**Comment by jkeck** [comment]
I added two commits to update the stacked logo svg and to address the logo + hamburger wrapping issue.

🍔 

new-mobile-and-icon

File: app/views/shared/_top_navbar.html.erb [hunk 1]
```diff
@@ -5,8 +5,8 @@
       <span class="fa fa-bars"></span>
     </button>
     <%= link_to 'http://library.stanford.edu' do %>
```
Comment (cbeer) [inline app/views/shared/_top_navbar.html.erb]: This should probably be an HTTPS link now, yeah?

Comment (jkeck) [inline app/views/shared/_top_navbar.html.erb]: Taking a quick look over the app there are a lot of http references.  Going to create a ticket about updating all of them holistically.

Comment (cbeer) [inline app/views/shared/_top_navbar.html.erb]: #1330

**Pull request merged**